_includes/performance.html

```diff
@@ -0,0 +1,91 @@
+<div class="full-width-bg component">
+  <div class="grid-wrapper">
+      <div class="width-3-12 width-12-12-m">
+        <img class="light-only" src="{{site.baseurl}}/assets/images/performance/icon-performance.svg" alt="Container image">
+        <img class="dark-only" src="{{site.baseurl}}/assets/images/performance/icon-performance-dark.svg" alt="Container image">
+      </div>
+      <div class="width-9-12 width-12-12-m">
+        <p class="intropara">Quarkus is engineered for efficiency. It  leverages build-time optimizations and a reactive core to achieve high throughput, fast startup times, low response latency, and minimal resource consumption. In other words, Quarkus is fast, and here is why; <strong>Reduced Memory Footprint, Fast startup Time and Higher Throughput</strong></p>
```

Comment:
+1, I like James revision. 

We had a few pictures in the draft at various stages, but I think maybe the one you mean is a mockup I made with graph lines. I think if we go that way we ought to get some 'real' data from @johnaohara and then abstract that into a graphic. It wouldn't be 'accurate' in the same way an actual graph would be, but it would have some foundation in a point-in-time reality. 

But I suggest we get the content up and then iterate on the visuals.